.travis.yml

```diff
@@ -2,21 +2,24 @@ language: python
 sudo: required
 dist: xenial
 python:
-- '2.7'
-- '3.5'
-- '3.6'
-- '3.7'
+  - '2.7'
+  - '3.5'
+  - '3.6'
+  - '3.7'
 env:
-- DJANGO_VERSION=1.8
-- DJANGO_VERSION=1.11
-- DJANGO_VERSION=2.0
-- DJANGO_VERSION=2.1
+  - DJANGO_VERSION=1.8
```

Comment:
wonder if we care too much about running on 1.8 ?

Reply:
Just want to make sure it works on all projects we have at least - I'll remove this as soon as humanly possible

Reply:
okey doke - good plan :)